src/core/geometry/path/clone.js

```diff
@@ -0,0 +1,3 @@
+const clone = path => Object.assign({}, path)
```

Comment:
A few comments that this is a shallow copy of the given would be good.

Reply:
That's a good point -- added jsdoc block.